control/responsible_role.go

```diff
@@ -1,15 +1,16 @@
 package control
 
 import (
-	"github.com/jbowtie/gokogiri/xml"
 	"errors"
+	"fmt"
 	"regexp"
 	"strings"
-	"fmt"
+
+	"github.com/jbowtie/gokogiri/xml"
 )
 
 // findResponsibleRole looks for the Responsible Role cell in the control table.
-func findResponsibleRole(ct *Table) (*responsibleRole, error) {
+func findResponsibleRole(ct *SummaryTable) (*responsibleRole, error) {
```

Comment:
since we aren't using `Table` to represent `Control Table` anymore, could we rename `ct` to `st`?